### ISSUE TITLE
Admin 5..0.5/6 Admin (React mode) add instance not working if adapter was loaded from git and not npm!

### ISSUE THREAD

**Issue by frankjoke** [ioBroker/ioBroker.admin#741]
If I load an adapter via Git repo I cannot use the (+) in admin to add an instance to the adapter. 
If I use iobroker cli it works.

**Comment by Apollon77** [comment]
Mabe needed to reload?

**Comment by frankjoke** [comment]
No, tested on multiple instances and even loading admin from various browsers.
If you switch to non-react mode it works.

**Comment by GermanBluefox** [comment]
Can you provide which commands are called in Admin5 and Admin4 (in output window you can see that). Be sure, that you use at least 5.0.13 of admin

**Comment by frankjoke** [comment]
Hi again! I just updated test systems to js-controller 3.3.5 and admin to 5.0.14 and it seems to work now on all updated systems.

Before when I added an adapter via 'Beliebig' from git (my beta versions...) add instance did not work and show up the menue for instance number at all. Now it works!

Ok, there is still the lag that admin needs a restart to show adapter but this is also the case for other installed adapters.

p.s.: I do have another problem with ALL admins and js-controller 3.x: The green/red/yellow dots do not show correct state always, they also turn red/yellow in between even adapter is running normally. I need to check still one thing I changed on most instances the `statisticsInterval` to 60000 (from normal 15000) and I need to check if this is the reason for the wrong behaviour. Admin itself also shows sometimes red dot, also on admion 4.2.1 (as well as 5.0.x both in react and normal mode). I therefore believe it's not an admin-problem.
